src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheckTest.java

```diff
@@ -123,4 +123,20 @@ public void testIgnoreOverriddenMethodsFalse()
                 getPath("InputParameterNumber2.java"), expected);
     }
 
+    @Test
+    public void testIgnoreAnnotatedBy() throws Exception {
```

Comment:
Please add another test, showing that we can support fully qualified names on annotations, like `com.puppycrawl.tools.checkstyle.checks.sizes.parameternumber.MyAnno`.

Reply:
Should this be the case when the fully qualified name is specified in the property, or it should be able to derive it from just `MyAnno` (if this is the case, it isn't supported yet)

Reply:
>Should this be the case when the fully qualified name is specified in the property

Yes, thanks for asking.

>or it should be able to derive it from just MyAnno

No, we should try to keep this check as simple as possible, this would probably involve looking at the package, imports, etc.

Reply:
Also, it would be ideal to add an example with a fully qualified name to the docs to make it obvious that we need to qualify the name in the property to have a match.

Reply:
Done.